tests/playground.html

```diff
@@ -260,6 +265,57 @@
       'removeDynamicOption', Blockly.TestBlocks.removeDynamicDropdownOption_);
 }
 
+function setRenderDebugOptionCheckboxState(overrideOptions) {
+  Blockly.blockRendering.Debug.config = overrideOptions || {};
+  if(!overrideOptions) return;
+  var renderDebugOptionsListEl = document.getElementById('renderDebugOptions');
+  var renderDebugOptionInputs =
+      renderDebugOptionsListEl.getElementsByTagName('INPUT');
+  for (var optionInput of renderDebugOptionInputs) {
+    var optionId = optionInput.getAttribute('id');
+    var checkboxMatch = optionId.match('^RenderDebug(.*)Check$');
+    if (!checkboxMatch) continue;
+    var optionName = checkboxMatch[1];
+    optionInput.checked = !!overrideOptions[optionName];
+  }
+}
+
+function updateRenderDebugOptions() {
+  var renderDebugOptionsListEl = document.getElementById('renderDebugOptions');
+  var renderDebugOptionInputs =
+      renderDebugOptionsListEl.getElementsByTagName('INPUT');
+  var debugOptions = {};
+  for (var optionInput of renderDebugOptionInputs) {
+    var optionId = optionInput.getAttribute('id');
+    var checkboxMatch = optionId.match('^RenderDebug(.*)Check$');
+    if (!checkboxMatch) continue;
+    var optionName = checkboxMatch[1];
+    debugOptions[optionName] = optionInput.checked;
+  }
+  Blockly.blockRendering.Debug.config = debugOptions;
+  sessionStorage.setItem(
+      'blockRenderDebugOptions',JSON.stringify(debugOptions));
+}
```

Comment:
Would it also be possible to re-render the workspace when an option is updated: 
``workspace.render()``

Reply:
I would also prefer for the workspace to be re-rendered. I've added the call.